runtime/src/weights/pallet_dactr.rs

```diff
@@ -0,0 +1,115 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Autogenerated weights for da_control
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2023-07-01, STEPS: `50`, REPEAT: 25, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! HOSTNAME: `ip-172-31-5-24`, CPU: `Intel(R) Xeon(R) Platinum 8175M CPU @ 2.50GHz`
+//! EXECUTION: Some(Native), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
+
+// Executed Command:
+// ./target/release/data-avail
+// benchmark
+// pallet
+// --chain=dev
+// --steps=50
+// --repeat=25
+// --log=warn
+// --execution=native
```

Comment:
I think we should generate the weights on wasm execution(worst case)

Reply:
I did not rerun benchmarks yet, it will be done on a separate PR sor this is the old one, you can see we don't have proof size on this file.

Reply:
What do you mean by ? I think we should generate the weights on wasm execution(worst case)